### ISSUE TITLE
moving between tabs with gt and gT

### ISSUE THREAD

**Issue by jerabaul29** [onivim/oni2#1642]
One command i use a lot is moving between tabs with gt and gT. It does not seem to work now. Any way to get that? :)

**Comment by CrossR** [comment]
I think that this in your `keybindings.json` (Ctrl-Shift-P => Open Keybindings) should work, though I've not tested it:

```json
  { "key": "gt", "command": "workbench.action.nextEditor", "when": "normalMode" },
  { "key": "gT", "command": "workbench.action.previousEditor", "when": "normalMode" }
```

The broader fix will be #592, so these things won't be needed, and you can just opt into normal vim behaviour.

(If that keybinding doesn't work, we should fix it, so that it works for people in the mean time, whilst we implement #592.)

**Comment by jerabaul29** [comment]
Using the bindings as you suggest work, many thanks. Wonder if this should be set by default when downloading the image, so that vim users are not surprised?

Looking forward to seeing #592 implemented fully :) .
